docs-conceptual/azps-3.5.0/azureps-in-docker.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Using Azure PowerShell in Docker
+description: How to use Azure PowerShell that is preinstalled in a Docker image.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 02/26/2020
+---
+
+# Using Azure PowerShell in Docker
+
+We are publishing Docker images with Azure PowerShell preinstalled. This article shows you how to
+get started using Azure PowerShell in the Docker container.
+
+## Finding available images
+
+The released images require Docker 17.05 or newer. It is also expected that you are able to run
+Docker without `sudo` or local administrative rights. Please follow Docker's official
+[instructions][install] to install `docker` correctly.
+
+The release containers are built from the official PowerShell containers, then the Az module added
+as a layer.
+
+The latest stable image includes:
+
+- PowerShell version 6.2.4
+- Azure PowerShell Az modules version 3.5.0
```

Comment:
We will update the image at each build of the Az module hence the following suggestion.

```suggestion
- Azure PowerShell most current Az module 
```